domain-management/src/main/resources/org/jboss/as/domain/management/_private/LocalDescriptions.properties

```diff
@@ -87,6 +87,7 @@ core.access-control.role-mapping.principal.add=Add a new principal to role mappi
 core.access-control.role-mapping.principal.remove=Remove an existing principal to role mapping.
 core.access-control.role-mapping.principal.type=The type of the Principal being mapped, either 'group' or 'user'.
 core.access-control.role-mapping.principal.realm=An optional attribute to map based on the realm used for authentication.
+core.access-control.role-mapping.principal.realm.deprecated=This attribute is no longer used once migrated to Elytron based security.
```

Comment:
Once what is migrated?

Perhaps "once the configuration is migrated". Or as you used in the xsd "once management security is migrated".